ads/vdo.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {validateData, writeScript} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function vdo(global, data) {
+  /*eslint "google-camelcase/google-camelcase": 0*/
+  console.log(global, data);
```

Comment:
Do you need this for production?

Reply:
Thanks for identifying this piece of code. We would like to remove this for production purposes. This was being used by us during development stage. Can you please remove this from your end? That would be great :D

Reply:
I have updated the code myself. No need for further modifications. Please have a final look. Thanks !